ledger/ledger-api-client/src/main/scala/com/digitalasset/ledger/client/services/admin/UserManagementClient.scala

```diff
@@ -116,7 +116,7 @@ object UserManagementClient {
       metadata: com.daml.ledger.api.v1.admin.object_meta.ObjectMeta
   ): domain.ObjectMeta = {
     domain.ObjectMeta(
-      // TODO um-for-hub: Ledger-api-client shouldn't need to know how to parse resourceVersion. (ledger-api-domain probably shouldn't be used here)
+      // It's unfortunate that a client is using the server-side domain ObjectMeta and has to know how to parse the resource version
```

Comment:
I don't see an easy way how to solve it so instead (1) leaving this node and (2) rearranging party related in the domain object to decrease the entropy